src/Sentry.Unity.Editor/Android/DebugSymbolUpload.cs

```diff
@@ -182,15 +190,23 @@ public void RemoveUploadFromGradleFile()
     {
         _logger.LogDebug("Attempting to remove the previous upload task from the gradle project.");
         var gradleBuildFile = LoadGradleScript();
-        if (!gradleBuildFile.Contains("sentry.properties"))
+        if (!gradleBuildFile.Contains(SymbolUploadTaskStartComment))
         {
             _logger.LogDebug("No previous upload task found.");
             return;
         }
 
-        var regex = new Regex(Regex.Escape(SymbolUploadTaskStartComment) + ".*" + Regex.Escape(SymbolUploadTaskEndComment), RegexOptions.Singleline);
+        var pattern = Regex.Escape(SymbolUploadTaskStartComment) + @"\r?\n.*?\r?\n" + Regex.Escape(SymbolUploadTaskEndComment);
+        var regex = new Regex(pattern, RegexOptions.Singleline);
```

Comment:
This does not look right, why is the change necessary? `RegexOptions.Singleline` already makes regex match `\n` by '.'